lib/readline.js

```diff
@@ -216,6 +217,7 @@ function Interface(input, output, completer, terminal) {
 
     this.history = [];
     this.historyIndex = -1;
+    console.log('Yes initialized....\n\n\n', this.history);
```

Comment:
☝️ is that debug?

Reply:
My bad, yes did for debugging. Will remove them and update the PR with test cases :)